pyogrio/tests/test_geopandas_io.py

```diff
@@ -328,11 +330,38 @@ def test_write_datetime_mixed_offset(tmp_path, use_arrow):
     assert_series_equal(result["dates"], utc_col)
 
 
+@pytest.mark.parametrize("ext", [ext for ext in ALL_EXTS if ext != ".shp"])
 @pytest.mark.filterwarnings(
     "ignore: Non-conformant content for record 1 in column dates"
 )
```

Comment:
For the non-tz case, this filterwarnings might not be needed? 
(don't remember exactly what was the reason for the non-conformity, but potentially the `-05:00` was causing that)

Reply:
(indeed the offset: https://github.com/OSGeo/gdal/issues/8460#issuecomment-1732382430)